velero-plugins/main.go

```diff
@@ -53,74 +55,56 @@ func main() {
 		RegisterRestoreItemAction("openshift.io/20-SCC-restore-plugin", newSCCRestorePlugin).
 		RegisterRestoreItemAction("openshift.io/21-role-bindings-restore-plugin", newRoleBindingRestorePlugin).
 		RegisterRestoreItemAction("openshift.io/22-cluster-role-bindings-restore-plugin", newClusterRoleBindingRestorePlugin).
-
 		Serve()
 }
-
```

Comment:
Let's leave these line breaks. Looks better in my opinion.

Reply:
So, need to remove line break, right?

Reply:
In this file let's add the line breaks back in. For some reason they were removed here

Reply:
Added